src/editpage/components/assistance/SettingsModal.vue

```diff
@@ -192,14 +192,19 @@ export default {
     },
 
     exportLogs: function() {
-      const {dialog} = require('electron').remote;
+      const electron = require('electron');
+      const {dialog} = electron.remote;
       const options = {defaultPath: 'logs.zip'};
       const targetDir = dialog.showSaveDialogSync(options);
-      const electron = require('electron');
       const sourceDir = (electron.app || electron.remote.app).getPath(
           'userData') + '\\activity-logs';
       const zipper = require('zip-local');
-      zipper.sync.zip(sourceDir).compress().save(targetDir + '.zip');
+      zipper.zip(sourceDir, function(error, zipped) {
+        if (!error) {
+          zipped.compress();
+          zipped.save(targetDir);
```

Comment:
Before you called `save(...)` on the result of `compress()` I assume this also works?